tezos/new_context/src/timings.rs

```diff
@@ -49,10 +69,545 @@ pub fn context_action(
     // TODO - Bruno: it is possible to avoid these conversions by borrowing the internal
     // &str directly. Since we want this function to add as little overhead as possible
     // investigate doing that later once everything is working properly.
-    let action_name: String = action_name.to_rust(rt);
+    let name: String = action_name.to_rust(rt);
     let key: Vec<String> = key.to_rust(rt);
-    let _ = (action_name, key, irmin_time, tezedge_time);
+    // let _ = (action_name, key, irmin_time, tezedge_time);
```

Comment:
You can remove this leftover comment here.

About the comment above, we cannot use the OCaml `str` now because we are sending it to another thread, but what can be done is to just compare the bytes (`action_name.as_str()`) and have an enum for the action type to which it is mapped (`"set" => ContextAction::Set`), that way we avoid the String allocation here.